wire-ios-data-model/Source/Model/Label/Label.swift

```diff
@@ -104,7 +104,7 @@ public class Label: ZMManagedObject, LabelType {
         // (1) the persistent store coordinator and the SQLite engine, and (2) touch the file system.
         // Looping through all objects in the context is way cheaper, because it does not involve (1)
         // taking any locks, nor (2) touching the file system.
-        return context.performAndWait {
+        context.performAndWait {
```

Comment:
Is this supposed to be changed to `perform` as well?

Reply:
No, this is not an async method.